### PR TITLE
[AAP-8173] Host Jobs List

### PR DESCRIPTION
**Pull request by prat98**
This PR addresses AAP-8173 which entails adding a jobs list for the hosts section which has all the jobs running on the host

### CODE REVIEW

File: frontend/awx/resources/inventories/inventoryHostsPage/InventoryHostFacts.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+export function InventoryHostFacts() {
+  const { t } = useTranslation();
+  const params = useParams<{ id: string; inventory_type: string; host_id: string }>();
+
+  const { data: facts } = useGet<object>(awxAPI`/hosts/${params.host_id ?? ''}/ansible_facts`);
+
+  return (
+    <PageDetails>
+      <PageDetailCodeEditor label={t('Facts')} value={JSON.stringify(facts) || '{}'} />
```
Comment (skochay330) [inline frontend/awx/resources/inventories/inventoryHostsPage/InventoryHostFacts.tsx:16]: The old UI allows you to toggle between YAML and JSON formats here- is that something we want to carry over to the new UI as well?

File: frontend/awx/resources/inventories/inventoryHostsPage/hooks/useHostsJobsColumns.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      header: t('Finish Time'),
+      cell: (job: UnifiedJob) => {
+        return <DateTimeCell value={job.finished} format="date-time" />;
+      },
+      sort: 'finished',
+    }),
+    [t]
+  );
+
+  const tableColumns = useMemo<ITableColumn<UnifiedJob>[]>(
```
Comment (skochay330) [inline frontend/awx/resources/inventories/inventoryHostsPage/hooks/useHostsJobsColumns.tsx:73]: I noticed you didn't add the expanded row on the jobs list page- just want to make sure that was intentional

**Review by skochay330 — COMMENTED** [review]
Looks good, just a couple discrepancies between old and new UI

File: frontend/awx/resources/inventories/inventoryHostsPage/hooks/useHostsJobsColumns.tsx (hunk 2 above)
Comment (prat98) [reply frontend/awx/resources/inventories/inventoryHostsPage/hooks/useHostsJobsColumns.tsx:73]: Missed that just added the extra columns now

File: frontend/awx/resources/inventories/inventoryHostsPage/InventoryHostFacts.tsx (hunk 1 above)
Comment (prat98) [reply frontend/awx/resources/inventories/inventoryHostsPage/InventoryHostFacts.tsx:16]: So what I've been seeing when comparing the old UI to the new UI in other pages. We don't have that toggle anymore. We are using the pageDetailCodeEditor component and I haven't seen a way to add a toggle to this component.


**Old UI**
Screenshot 2024-02-01 at 7 51 20 PM

**New UI**
Screenshot 2024-02-01 at 7 51 10 PM

**Review by skochay330 — APPROVED** [review]

**Pull request merged**